sdk/formrecognizer/azure-ai-formrecognizer/migration-guide.md

```diff
@@ -0,0 +1,255 @@
+
+## Table of contents
+
+- [Guide for migrating to `azure-ai-formrecognizer (4.0.0-beta.1)` from `azure-ai-formrecognizer (3.0.x)`](#guide-for-migrating-to-azure-ai-formrecognizer-400-beta1-from-azure-ai-formrecognizer-30x)
+    - [Table of contents](#table-of-contents)
+    - [Migration benefits](#migration-benefits)
+        - [Cross Service SDK improvements](#cross-service-sdk-improvements)
+    - [Important changes](#important-changes)
+        - [Instantiating clients](#instantiating-clients)
+        - [Analyze Documents using a Prebuilt Model](#analyze-documents-using-a-prebuilt-model)
+        - [Build a custom document analysis model](#build-a-custom-document-analysis-model)
+        - [Analyze Documents using a Custom Model](#analyze-documents-using-a-custom-model)
+        - [Manage models](#manage-models)
+    - [Additional samples](#additional-samples)
+
+## Migration benefits
+
+A natural question to ask when considering whether or not to adopt a new version or library is its benefits. As Azure has matured and been embraced by a more diverse group of developers, we have been focused on learning the patterns and practices to best support developer productivity and to understand the gaps that the Java
+client libraries have.
+
+There were several areas of consistent feedback expressed across the Azure client library ecosystem. The most important is that many developers have felt that the learning curve was difficult, and the APIs did not offer a good, approachable, and consistent onboarding story for those learning Azure or exploring a specific Azure service.
+
+To improve the development experience across Azure services, a set of uniform [design guidelines][Guidelines] was created for all languages to drive a
+consistent experience with established API patterns for all services. A set of [Java design guidelines][GuidelinesJava] was introduced to ensure that Java clients have a natural and idiomatic feel with respect to the Java ecosystem. Further details are available in the guidelines
+for those interested.
+
+Aside from the benefits of the new design mentioned above, the `azure-ai-formrecognizer (4.0.0-beta.1)` replaces the multiple API design and adds a unified method to analyze documents with the `beginAnalyzeDocument` method and managing of operations in your form recognizer account.This version of the client library defaults to the 2021-09-30-preview version of the service. Refer to the [README][README] for more information about analyzing documents.
+
+### Cross Service SDK improvements
+
+The modern Form Recognizer client library also provides the ability to share in some of the cross-service improvements made to the Azure development experience, such as
+
+- A unified method for analyzing analyzing text and structured data from your documents across each of the client libraries.
```

Comment:
```suggestion
- A unified method for analyzing text and structured data from your documents across each of the client libraries.
```

Also think about using "extract" here instead of "analyze".